src/app/pages/LandingPage/components/CryptocurrencyPrices/index.tsx

```diff
@@ -184,19 +178,23 @@ export const Row: React.FC<IRowProps> = ({
           </strong>
         </td>
 
-        <td className="tw-text-right tw-whitespace-nowrap">
+        <td className="tw-text-left tw-text-sm tw-font-inter tw-whitespace-nowrap">
           {numberToUSD(lastPrice || 0)}
         </td>
 
-        <td className={'tw-text-right tw-whitespace-nowrap'}>
+        <td className={'tw-text-left tw-text-sm tw-whitespace-nowrap'}>
           <PriceChange value={price24h} />
```

Comment:
![UI issue 5](https://user-images.githubusercontent.com/56412037/144450794-1f36e082-7d35-4879-a89a-7307d19c3656.JPG)
In UI/UX, these 2 (24h, 7d) colors are different. They are not white color.

Reply:
Maybe it's white as the values are 0.

Reply:
So, the color code should be like this:
- White if `0`
- Red if negative values
- Green if positive values